trace_decoder/tests/test_b19807080.rs

```diff
@@ -0,0 +1,66 @@
+//! This test aims at ensuring that the decoder can properly parse a block trace
+//! received from Jerigon into zkEVM `GenerationInputs`, which the prover can
+//! then pick to prove each transaction in the block independently.
+//!
+//! This test only `simulates` the zkEVM CPU, i.e. does not generate STARK
+//! traces nor generates proofs, as its purpose is to be runnable easily in the
+//! CI even in `debug` mode.
+//!
+//! The tested block is this one: <https://etherscan.io/block/19807080>.
+
+use std::time::Duration;
+
+use evm_arithmetization::prover::testing::simulate_execution;
+use plonky2::field::goldilocks_field::GoldilocksField;
+use plonky2::util::timing::TimingTree;
+use pretty_env_logger::env_logger::{try_init_from_env, Env, DEFAULT_FILTER_ENV};
+use serde::{Deserialize, Serialize};
+use trace_decoder::{
+    processed_block_trace::ProcessingMeta,
+    trace_protocol::BlockTrace,
+    types::{CodeHash, OtherBlockData},
+};
+
+fn resolve_code_hash_fn(_: &CodeHash) -> Vec<u8> {
+    todo!()
```

Comment:
I this related to the " blob versioned hashes sanity check" ?

Reply:
And I will just reveal my ignorance and ask: how is it possible that this `todo!()` is never actually called?

Reply:
This is unrelated, see https://github.com/0xPolygonZero/zk_evm/issues/159